.travis/flake8.sh

```diff
@@ -2,7 +2,7 @@
 # Run pep8 on all .py files in all subfolders
 
 tmpafter=$(mktemp)
-find ./libiocage/ -name \*.py -exec flake8 --ignore=E203,W391 {} + | grep -v "./libiocage/__init__.py" > ${tmpafter}
+find ./libiocage/ -name \*.py -exec flake8 --ignore=E203,E241,W391 {} + | grep -v "./libiocage/__init__.py" > ${tmpafter}
```

Comment:
What is `E241` good for? The documentation says `E241 (*) | multiple spaces after ‘,’`, but I can't spot why we need to ignore this rule.

Reply:
I agree. I added the ignore for spaces after `:` so we shouldn't need to add another.

Reply:
without this,
```
./libiocage/lib/Jail.py:72:23: E241 multiple spaces after ':'
./libiocage/lib/Jail.py:73:22: E241 multiple spaces after ':'
./libiocage/lib/Jail.py:74:23: E241 multiple spaces after ':'
./libiocage/lib/Jail.py:76:22: E241 multiple spaces after ':'
```
i've basically just followed @skarekrow's lead of how to indent… structures.